Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/core/RecoveryCertificateWrapper.kt

```diff
@@ -13,7 +13,7 @@ data class RecoveryCertificateWrapper(
 
     val isUpdatingData = container.isUpdatingData
 
-    val testCertificate: RecoveryCertificate? by lazy {
-        container.toRecoveryCertificate()
```

Comment:
there are no value sets for recovery

Reply:
using the current VaccinationValueSet, need to reconsider the naming since it is now used for two certificates